.github/ISSUE_TEMPLATE/release_checklist.md

```diff
@@ -24,8 +24,8 @@ assignees: ''
 - [ ] run `src/gmt_make_*.sh` to update some .c and .h files
 - [ ] run `admin/gs_check.sh` to test if latest ghostscript version works
 - [ ] update [changelog](/doc/rst/source/changes.rst)
-- [ ] update INSTALL.md
-- [ ] check if there are any warnings when build the documentation
+- [ ] update [INSTALL.md](/INSTALL.md) (only needed for major releases)
```

Comment:
Perhaps change it to:

> check install instructions in [INSTALL.md](/INSTALL.md)

Sometimes, these package managers may make changes, which makes the install instructions out-of-date. I feel it would be better to review the instructions (no need to test them) before each minor release.

Reply:
Sounds good, updated in https://github.com/GenericMappingTools/gmt/pull/5113/commits/de4ad4a652481864492c51b3c6417116a5076df9